README.md

```diff
@@ -152,10 +173,56 @@ dashboard. On your development environment,
 make sure to use test keys. If you don't, you're
 going to be charged real money!
 
-#### Testing
+### Sripe keys setup
+
+#### Get stripe keys:
+
+Go to [stripe](https://stripe.com/en-br), create an account or just log in with you already have one. Acess the stripe dashboard and copy both publishable and secret keys.
+> make sure to use test keys. If you don't, you're
+going to be charged real money!
+
+![Screenshot from 2022-02-24 13-28-18](https://user-images.githubusercontent.com/44117439/155566057-97ab2370-7ed1-43ab-91fc-cdd4cb340390.png)
```

Comment:
If possible, could you adjust the highlight color over "Publishable key" and "Secret key" to something with more contrast from the text? People who have certain vision disabilities likely wouldn't be able to read those words.